docs/Technical-Documentation/tech-memos/priotitized-errors/prioritized-errors.md

```diff
@@ -0,0 +1,101 @@
+# TDP Prioritized Parser Errors
+
+**Audience**: TDP Software Engineers <br>
+**Subject**:  Prioritized Errors <br>
+**Date**:     October 20, 2024 <br>
+
+## Summary
+This technical memorandum provides a suggested path to implement a set of new requirements OFA has generated to alleviate the sheer number of parser errors generated during a STT's data submission. OFA has indicated, while still necessary for admins to see, some errors are of a lower priority for STTs to view and even prevent them from using the submission history reports to correct their submissions. Thus, the OFA team has requested that a "priority" be assigned to parser errors so that the report STTs receive is filtered down to only the highest priority errors that must be fixed. Regardless of how errors are prioritized, OFA admins will still retain the ability to generate the unfiltered error report for further, more in-depth investigation if a necessary situation presents itself.
```

Comment:
```suggestion
This technical memorandum provides a suggested path to implement a set of new requirements OFA has generated to alleviate the sheer number of parser errors generated during a STT's data submission. OFA has indicated that some errors are of a lower priority for STTs to review and correct. Thus, the OFA team has requested that  "critical" be assigned to parser errors so that the report STTs receive is filtered down to only the critical errors that must be reviewed and fixed. Regardless of how errors are prioritized, STTs will still retain the ability to see a summary of all errors detected in the error report.
```